src/main/java/jp/classmethod/aws/gradle/lambda/AWSLambdaMigrateFunctionTask.java

```diff
@@ -23,6 +23,8 @@
 import java.nio.channels.FileChannel;
 import java.util.Map;
 
+import com.amazonaws.services.lambda.model.*;
```

Comment:
Avoid using wildcards please

Reply:
sure. it's fixed in the following commit.